css/style.css

```diff
@@ -1,6 +1,163 @@
 /* General page styles */
 
-/* We should use a mobile first approach having out breack point at 768px */
+/* We should use a mobile first approach having out brake point at 768px */
+
+* {
+	box-sizing: border-box;
+}
+
+body {
+	margin: 0;
+	padding: 0;
+}
+
+.container {
+	position: relative;
+	margin: auto;
+	max-width: 960px;
+	height: 100%;
+}
+
+.main-nav {
+	position: fixed;
+	top: 0;
+	z-index: 2;
+	width: 100%;
+	height: 75px;
+	overflow-x: hidden;
+	transition: 0.5s;
+}
+
+.colored__main-nav {
+	background-color: rgba(0, 60, 143, 0.8);
+}
+
+.transparent__main-nav {
+	background-color: rgba(0, 60, 143, 0);
+}
+
+.main-nav__logo {
+	position: absolute;
+	top: 11px;
+	left: 20px;
+}
+
+.main-nav__logo img {
+	width: 89px;
+}
+
+.main-nav__burguer {
+	position: absolute;
+	top: 20px;
+	right: 20px;
+	cursor: pointer;
+}
+
+.main-nav__burguer div {
+	width: 40px;
+	height: 7px;
+	background-color: white;
+	transition: 0.5s;
+}
+
+.burguer__bar-1,
+.burguer__bar-2 {
+	margin-bottom: 7px;
+}
+
+.burguer-animation .burguer__bar-1 {
+	transform: rotate(-45deg) translate(0px, 14px);
+	transform-origin: center center;
+	margin: 0;
+}
+
+.burguer-animation .burguer__bar-2 {
+	opacity: 0;
+}
+
+.burguer-animation .burguer__bar-3 {
+	transform: rotate(45deg) translate(0px, -14px);
+	transform-origin: center center;
+}
+
+.main-nav__menu-items {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	position: absolute;
+	top: 0;
+	z-index: -1;
+	width: 100%;
+	height: 0px;
+	margin: 0px;
+	padding: 0px;
+	opacity: 0;
+	list-style-type: none;
+	overflow: hidden;
+}
+
+.main-nav__menu-items a {
+	color: white;
+	font-family: rooboto, sans-serif;
+	font-size: 30px;
+	font-weight: normal;
+	letter-spacing: normal;
+	line-height: 2;
+	text-decoration: none;
+}
+
+.main-nav-overlay__animation {
+	height: 100%;
+}
+
+.main-nav__menu-items__animation {
+	height: 100%;
+	opacity: 1;
+}
+
+/* =========== MAIN-NAV DESKTOP ============ */
+
+@media screen and (min-width: 768px) {
```

Comment:
the min-width is 768px. but we are working with  960px. can you explain that?

Reply:
Good catch, the media querie needs to be adjusted to 960px

Reply:
Do we have a new brake point for responsive view? Isn't it 768px?

Reply:
it was been changed almost a month a go, and has been mentioned constantly in meetings and PRs.

Reply:
Fixed, thank you.